dune-precice/dune-precice/couplinginterface.hh

```diff
@@ -86,22 +83,22 @@ namespace Dune::preCICE {
         int iter;
       } previous_solver_state_;
      
-
-
     public:
               
       CouplingInterface(const ParameterClass &parameters, 
                         const BlockVector<FieldVector<bool, dim>> &is_coupling_boundary,
                         int mpi_rank, int mpi_size);
 
       template <typename PolynomialBasis>
-      double initialize(const PolynomialBasis &polynomial_basis);
+      void initialize(const PolynomialBasis &polynomial_basis);
+
+      double get_max_time_step_size() const;
        
-      void read_blockvector_data(VectorType &dune_data);
+      void read_blockvector_data(VectorType &dune_data, double dt);
```

Comment:
It is probably a good idea to rename these function to be similar to the new preCICE API. Older names will confuse future users.